data_summaries/Open-Platypus.json

```diff
@@ -0,0 +1,556 @@
+        "ArXiv URL": "https://arxiv.org/abs/2209.09513",
+        "Semantic Scholar Corpus ID": 252383606,
+        "Collection": "Open-Platypus",
+        "Collection URL": "https://platypus-llm.github.io",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "Chain-of-Thought",
+            "Explanation",
+            "Question Answering"
+        ],
+        "Text Sources": [
+            "Books",
+            "Exams"
+        ],
+        "Model Generated": [],
+        "Format": [
+            "Chain-of-Thought",
+            "Few-shot",
+            "Zero-shot"
+        ],
+        "Human Annotation": "Yes",
+        "Derived from Datasets": [],
+        "Creators": [
+            "UCLA",
+            "Arizona State University",
+            "Allen Institute for AI"
+        ],
+        "Licenses": [
+            {
+                "License": "CC BY-NC-SA 4.0",
+                "License URL": "https://github.com/lupantech/ScienceQA/blob/main/LICENSE-DATA"
+            }
+        ],
+        "License Notes": "Both the paper and data license on GitHub are Attribution-NonCommercial-ShareAlike 4.0 International",
+        "License Verified By": "Cole Hunter",
+        "Dataset Filter IDs": [
+            "scienceqa"
+        ]
+    },
+    "op-theoremqa": {
+        "Unique Dataset Identifier": "op-theoremqa",
+        "Dataset Name": "theorem-qa",
+        "Paper Title": "TheoremQA: A Theorem-driven Question Answering Dataset",
+        "Dataset URL": "https://huggingface.co/datasets/wenhu/TheoremQA",
+        "GitHub URL": "https://github.com/wenhuchen/TheoremQA",
+        "Hugging Face URL": "https://huggingface.co/datasets/wenhu/TheoremQA",
+        "Papers with Code URL": "https://paperswithcode.com/dataset/theoremqa",
+        "ArXiv URL": "https://arxiv.org/abs/2305.12524",
+        "Semantic Scholar Corpus ID": 258833200,
+        "Collection": "Open-Platypus",
+        "Collection URL": "https://platypus-llm.github.io",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "Logical and Mathematical Reasoning",
+            "Question Answering"
+        ],
+        "Text Sources": [
+            "Books",
+            "human",
+            "OpenAI GPT-4",
+            "undisclosed web"
+        ],
+        "Model Generated": [
+            "OpenAI GPT-4"
+        ],
+        "Format": [
+            "Chain-of-Thought",
+            "Program of Thoughts"
+        ],
+        "Human Annotation": "Yes",
+        "Derived from Datasets": [],
+        "Creators": [
+            "UCLA",
+            "UC Santa Barbara",
+            "University of Waterloo"
+        ],
+        "Licenses": [
+            {
+                "License": "MIT License",
+                "License URL": "https://huggingface.co/datasets/wenhu/TheoremQA"
+            }
+        ],
+        "License Notes": "",
```

Comment:
Do we know the dataset is MIT licensed because they collect it and put the license themselves? Just checking if its a case that they just slapped MIT  on top of data another data source that could have its own license?

Reply:
They used GPT-4 to generate university level theorems in STEM subfields and then a team of domain experts refined and reviewed the data. I have made the appropriate changes!